docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -40,8 +40,21 @@ To create an automated process with user tasks, take the following steps:
 ![user task example](./img/user-task-example.png)
 9. Append a gateway to your user task by dragging it onto the dashboard from the palette on the left side of the screen, or by clicking on the user task `Decide what's for dinner` and clicking on the element you'd like to create next. In this case, we've selected the diamond icon to create a gateway.
 10. Create two sequence flows (represented by the arrows) from the gateway and two new user tasks based on what the user decides to eat. In this case, we've named ours `Prepare chicken` and `Prepare salad`.
+   - Note that the sequence flows require [expressions](../components/concepts/expressions.md) to access variables from the form we'll create below to determine what to eat for dinner. To add an expression, click on the sequence flow to view the properties panel, and open the **Condition** tab to insert a conditional expression.
 11. Attach an end event to the two user tasks.
-![gateway example](./img/gateway-example-1.png)
+
+<div bpmn="./img/prepare-dinner.bpmn" />
```

Comment:
I fixed how this should look like in this commit: https://github.com/camunda-cloud/camunda-cloud-documentation/commit/c60ec0b90cab06bd7b50337cb862d9699ca7225f. But to make it work you have to cherry-pick the other commit from master

Reply:
Which commit needs to be removed from master?